website/versioned_docs/version-15.0.0/version.md

```diff
@@ -2,6 +2,35 @@
 title: Versions
 author: Jev Bjorsell
 ---
+# Taquito v15.0.1
```

Comment:
these changes seem a little funny to me 
these are for the version 15.0.0 docs? is that right that this should be updated with version 15.0.1?

Reply:
yeah, we don't really make a versioned_doc folder to index for patch releases. So I included it in 15.0.0 so it shows our users in both `Next` and current labels

Reply:
👍